scripts/removejujuservices.bash

```diff
@@ -1,19 +0,0 @@
-#!/bin/bash
```

Comment:
TBH: we should leave this here and get shellcheck to validate our shellscripts, then use a binary blob embedding tool to then ensure that it's served as a static file. 

See: https://github.com/gobuffalo/packr

Reply:
Restored it.